lib/tests/streamlit/elements/plotly_chart_test.py

```diff
@@ -107,3 +108,47 @@ def test_st_plotly_chart_sharing(self):
         self.assertEqual(el.plotly_chart.HasField("figure"), False)
         self.assertNotEqual(el.plotly_chart.url, "the_url")
         self.assertEqual(el.plotly_chart.use_container_width, False)
+
+    def callback():
+        pass
+
+    @parameterized.expand(
+        [
+            (True, True),
```

Comment:
This should likely be changed as we will only be using `True / False` OR `"rerun"/"ignore"`, aka not both options.